Facebook.iOS/source/Facebook/StructsAndEnums.cs

```diff
@@ -25,7 +25,8 @@ public enum ErrorCode : long
 		DialogUnavailable,
 		AccessTokenRequired,
 		AppVersionUnsupported,
-		BrowswerUnavailable
+		BrowserUnavailable,
+		BrowswerUnavailable = BrowserUnavailable
```

Comment:
Yep, a Facebook typo, I didn't see it either, they fixed it but without breaking apps.